### ISSUE TITLE
Running bitcoind in docker fails on a Raspberry Pi4

### ISSUE THREAD

**Issue by ArmanTheParman** [lnbook/lnbook#698]
**Describe the error**
Following along the docker guide.
Came to the command to run docker. (Sudo was missing, added that myself)
$ sudo docker run -it --name bitcoind lnbook/bitcoind
returned error
/usr/local/bin/bitcoind0entrypoint.sh line 5: /usr/local/bin/bitcoind: cannot execute binary file: Exec format error

Is this because I have an ARM chip not AMD? 
I will skip the rest of the chapter, but a shame, because I was learning a lot here.

**To Fix**
if the guide is not compatible with ARM (Raspberry Pi4 in my case), please state so early on.

**Comment by ckapilla** [comment]
yes this error is because of the ARM/AMD mismatch between your processor and the specific binary pointed to in the Dockerfile. Perhaps there should be a comment in the Dockerfile notifying people that if necessary they should edit the Dockerfile to specify the appropriate binary for the desired version and their operating system/processor, as are available here: https://bitcoin.org/bin/
